src/main/java/org/example/homework_nr_6/main.java

```diff
@@ -0,0 +1,13 @@
+package org.example.homework_nr_6;
+
+public class main {
+
+    public static void main(String[] args) {
+        Date myDate = new Date();
```

Comment:
Create a constructor for Date class and use constructor to load data below. After that use the methods below again in order to modify the data

Reply:
Fixed